packages/tooltip/src/Tooltip.tsx

```diff
@@ -306,30 +306,12 @@ function Tooltip({
           triggerProps[handler](e);
       }
     },
-    [handleClose, setOpen, tooltipNode],
+    [handleClose, setOpen, tooltipRef],
   );
 
   useEscapeKey(handleClose, { enabled: open });
 
-  const handleBackdropClick = useCallback(
-    (e: MouseEvent) => {
-      /**
-       * Close the tooltip iff the clicked target (e.target) is NOT the tooltip element
-       *
-       * This handler is added to the document.
-       * No need to check whether the click target is the trigger node
-       * since clicks on that element are stopped from propagating by the <Popover>
-       */
-      if (tooltipNode && !tooltipNode.contains(e.target as HTMLElement)) {
-        handleClose();
-      }
-    },
-    [handleClose, tooltipNode],
-  );
-
-  useEventListener('click', handleBackdropClick, {
-    enabled: open && triggerEvent === 'click',
-  });
+  useBackdropClick(handleClose, [tooltipRef], open);
```

Comment:
You didn't include `triggerEvent === 'click'`. It was previously `open && triggerEvent === 'click'`. If you're not including this can you make sure that this doesn't break `GuideCue`? That tooltip should stay open on background click.

Reply:
Good catch! I'll fix that